jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XMLStreamWriterFilter.java

```diff
@@ -53,6 +53,26 @@ public void writeStartDocument() throws XMLStreamException {
         writer.writeStartDocument();
     }
 
+    private boolean isWhiteSpace(char c){
+        // all char points between 0 and 31 excluding the allowed white spaces (9=TAB, 10=LF, 13=CR)
+        if (c >= 0 && c<=31 && c != 9 && c != 10 && c != 13)
```

Comment:
`c >= 0` - is it possible for char to be negative?
<sup>https://docs.oracle.com/javase/specs/jls/se12/html/jls-4.html#jls-4.2.1</sup>

Reply:
I don't think char will be negative
As per the doc https://docs.oracle.com/javase/specs/jls/se12/html/jls-4.html#jls-4.2.1
The values of the integral types are integers in the following ranges: 
For char, from '\u0000' to '\uffff' inclusive, that is, from 0 to 65535

Reply:
> I don't think char will be negative

So does it make sense to check `c >= 0`?